src/nspawn/nspawn.c

```diff
@@ -5385,13 +5482,16 @@ static int run_container(
                                 arg_kill_signal,
                                 arg_property,
                                 arg_property_message,
-                                arg_keep_unit,
                                 arg_container_service_name,
-                                arg_start_mode);
+                                arg_start_mode,
+                                flags);
                 if (r < 0)
                         return r;
 
         } else if (!arg_keep_unit) {
+                AllocateScopeFlags flags = 0;
```

Comment:
why not just assign ALLOCATE_SCOPE_ALLOW_PIDFD instead of 0 here, and drop the next line?

Reply:
Changed. I liked the symmetry of a `SET_FLAG` call for each flag.

Reply:
not actually changed?

Reply:
It was changed; this is the thing that ended up in the wrong commit.